### PR TITLE
Feature/civ 3305 reject and paid part 1

### PR DESCRIPTION
**Pull request by RosDiaf**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/modules/i18n/locales/en.json [hunk 1]
```diff
@@ -368,6 +372,11 @@
       "REJECT_DEADLINE": "You have until 4pm on {{ deadline }} to reject the claim.",
       "REJECT_PAID": "If you reject because you believe you've paid the money, the claimant has 28 days to tell you and the court whether they're proceeding with the claim.",
       "REJECT_WHOLE_CLAIM": "If you reject all of the claim, the claim may be transferred to a County Court hearing centre.",
+      "REJECT_ALL_CLAIM_TYPE": {
+        "ALREADY_PAID": "I've paid what I believe I owe",
```
Comment (laarmada) [inline src/main/modules/i18n/locales/en.json:376]: In A-Z order please

File: src/main/modules/i18n/locales/cy.json [hunk 2]
```diff
@@ -368,6 +372,11 @@
       "REJECT_DEADLINE": "Mae gennych tan 4pm ar {{ deadline }} i wrthod yr hawliad cyfan.",
       "REJECT_PAID": "Os ydych yn gwrthod oherwydd eich bod yn credu eich bod wedi talu'r arian, mae gan yr hawlydd 28 i ddweud wrthych chi a'r llys p'un a yw'n bwrw ymlaen â'r hawliad.",
       "REJECT_WHOLE_CLAIM": "Os ydych yn gwrthod yr hawliad cyfan, gall yr hawliad gael ei drosglwyddo i ganolfan wrandawiadau Llys Sirol.",
+      "REJECT_ALL_CLAIM_TYPE": {
```
Comment (laarmada) [inline src/main/modules/i18n/locales/cy.json:375]: In A-Z order please

**Review by laarmada — APPROVED** [review]
it´s possible to cover?: 
image

File: src/main/modules/i18n/locales/cy.json (hunk 2 above)
Comment (RosDiaf) [reply src/main/modules/i18n/locales/cy.json:372]: 1. **List re-ordered.**

2. **Unit test coverage 100%**
sonar-green

File: src/main/services/features/response/checkAnswers/responseSection/buildYourResponseToClaimSection.ts [hunk 3]
```diff
@@ -24,10 +39,12 @@ export const buildYourResponseToClaimSection = (claim: Claim, claimId: string, l
     summaryRows: [],
   });
 
-  yourResponseToClaimSection.summaryList.rows.push(summaryRow(t('PAGES.CHECK_YOUR_ANSWER.OWE_MONEY', { lng: getLng(lang) }), t(`COMMON.RESPONSE_TYPE.${claim.respondent1?.responseType}`, { lng: getLng(lang) }), yourResponseToClaimHref, changeLabel(lang)));
+  yourResponseToClaimSection.summaryList.rows.push(summaryRow(t('PAGES.CHECK_YOUR_ANSWER.OWE_MONEY', { lng: getLng(lang) }), t(`COMMON.RESPONSE_TYPE.${claim.respondent1.responseType}`, { lng: getLng(lang) }), yourResponseToClaimHref, changeLabel(lang)));
 
-  if (claim.respondent1?.responseType === ResponseType.PART_ADMISSION) {
+  if (claim.respondent1.responseType === ResponseType.PART_ADMISSION) {
```
Comment (laperla11) [inline src/main/services/features/response/checkAnswers/responseSection/buildYourResponseToClaimSection.ts:44]: you can use `claim.isPartialAdmission()` method here

File: src/main/services/features/response/checkAnswers/responseSection/buildYourResponseToClaimSection.ts [hunk 4]
```diff
@@ -24,10 +39,12 @@ export const buildYourResponseToClaimSection = (claim: Claim, claimId: string, l
...
     yourResponseToClaimSection.summaryList.rows.push(summaryRow(t('PAGES.CHECK_YOUR_ANSWER.RESPONSE_HAVE_YOU_PAID_THE_CLAIMANT', { lng: getLng(lang) }), t(`COMMON.${alreadyPaid}`, {lng: getLng(lang)}), yourPaymentAdmittedToClaimantHref, changeLabel(lang)));
+  } else if (claim.respondent1.responseType === ResponseType.FULL_DEFENCE) {
```
Comment (laperla11) [inline src/main/services/features/response/checkAnswers/responseSection/buildYourResponseToClaimSection.ts:46]: you can use  `claim. isFullDefence()` method here

**Review by laperla11 — APPROVED** [review]

File: src/main/services/features/response/checkAnswers/responseSection/buildYourResponseToClaimSection.ts (hunk 3 above)
Comment (RosDiaf) [reply src/main/services/features/response/checkAnswers/responseSection/buildYourResponseToClaimSection.ts]: Changed to claim.isPartialAdmission()

File: src/main/services/features/response/checkAnswers/responseSection/buildYourResponseToClaimSection.ts (hunk 4 above)
Comment (RosDiaf) [reply src/main/services/features/response/checkAnswers/responseSection/buildYourResponseToClaimSection.ts]: Changed to claim.isFullDefence()

File: src/main/modules/i18n/locales/en.json (hunk 1 above)
Comment (RosDiaf) [reply src/main/modules/i18n/locales/en.json:373]: List re-ordered.

**Pull request merged**